index.js

```diff
@@ -63,4 +63,5 @@ E.prototype = {
   }
 };
 
-module.exports = E;
+module.exports.default = E;
+module.exports.emitter = new E();
```

Comment:
Do you see any downside to always instantiating an object when this file is required?